### ISSUE TITLE
Videos with DTS won't Play

### ISSUE THREAD

**Issue by Err0r0815** [jellyfin/Swiftfin#496]
### Describe the bug

Hello,

if i try to play an video with DTS as audio track, it wont start. the screen stays black. if i fast forward, the time will synced to the server.
The server wont transcode it to ac3.
the "playinfo" says "file are compatible, direct play"

things i try to fix it:
1. encode only the audio to ac3 with ffmpeg to a new file => works
2. with the DTS audio activate "force direct play" = doesn´t work
3. with the DTS audio activate "force native player" = doesn´t work

i hope it helps to fix it

thanks and greetings :-)

### Application version

1.0.0 (Build 74)

### Where did you install the app from?

TestFlight

### Device information

Apple TV 4k 1st gen

### OS version

tvOS 15.5.1

### Jellyfin server version

10.8.1

**Comment by kikher** [comment]
I have the same problem, it only happens with DTS videos.
In my case, the iPhone app works and plays the video, but the Apple TV app doesn't play it, it stays black and it doesn't report any errors or problems.

**Comment by LePips** [comment]
This is probably due to our [device builder needing improvements](https://github.com/jellyfin/Swiftfin/issues/296) so that it can request a transcode to ac3.

**Comment by holow29** [comment]
I think VLCKit should handle DTS just fine - VLC app for tvOS does at least. I don't think it should need to transcode. (I've been trying to work on the device builder a bit.)

**Comment by LePips** [comment]
Sounds great! Since this is a big feature, would you mind making a draft PR so I can take a look?

**Comment by Err0r0815** [comment]
> I think VLCKit should handle DTS just fine - VLC app for tvOS does at least. I don't think it should need to transcode. (I've been trying to work on the device builder a bit.)

yeah, ich i use the VLC app on my apple tv, every video plays fine :-)

**Comment by zackify** [comment]
wow this is still not working for me haha

**Comment by JPKribs** [comment]
This should be resolved in Swiftfin 1.5 which is now available on the App Store!

Please reach out or reopen this issue if you are still experiencing trouble with this item.
